src/app/Http/Controllers/Admin/Swagger/LaravelLogController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers\Admin\Swagger;
+
+/**
+ * @OA\Schema(schema="LaravelLogBase"),
+ */
+class LaravelLogController extends Controller
+{
+    /**
+     * @OA\Get(path="/api/v1/logs/laravel", summary="Retrieves laravel log", description="Retrieve the laravel log if it exists.", operationId="getLaravelLog", tags={"rootServer"}, security={{"bmltToken":{}}},
+     *     @OA\Response(response=200, description="Returns when user is authenticated.",
+     *         @OA\MediaType(mediaType="text/plain", @OA\Schema(type="string"))
```

Comment:
Look at the "Response That Returns a File" section of https://swagger.io/docs/specification/describing-responses/.

Reply:
fixed, its cool the swagger ui generates a download link for you with the gzip file